x-pack/plugins/beats_management/server/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PathReporter } from 'io-ts/lib/PathReporter';
+import { isLeft } from 'fp-ts/lib/Either';
+import { KibanaRequest, Headers, Logger } from 'src/core/server';
+import {
+  BackendFrameworkAdapter,
+  FrameworkInfo,
+  FrameworkUser,
+  internalAuthData,
+  internalUser,
+  RuntimeFrameworkInfo,
+  RuntimeKibanaUser,
+} from './adapter_types';
+import { BeatsManagementConfigType } from '../../../../common';
+import { ILicense, LicensingPluginStart } from '../../../../../licensing/server';
+import { SecurityPluginSetup } from '../../../../../security/server';
+
+export class KibanaBackendFrameworkAdapter implements BackendFrameworkAdapter {
+  public readonly internalUser = internalUser;
+  public info: null | FrameworkInfo = null;
```

Comment:
Too many changes to keep file track, but this was `x-pack/legacy/plugins/beats_management/server/lib/adapters/framework/kibana_framework_adapter.ts`